authors/settings.py

```diff
@@ -96,16 +96,16 @@
 
 AUTH_PASSWORD_VALIDATORS = [
     {
-        'NAME': 'django.contrib.auth.password_validation.UserAttributeSimilarityValidator',
+        'NAME': 'django.contrib.auth.password_validation.UserAttributeSimilarityValidator',     # noqa E501
```

Comment:
To ignore an error that appears severally in one file, you can have a comment at the top e.g `# pylint:disable=invalid-name`